cli/src/main/java/rules/CheckTagsAreDefinedRule.java

```diff
@@ -0,0 +1,23 @@
+package rules;
+
+import org.eclipse.microprofile.openapi.models.OpenAPI;
+import org.openapitools.openapistylevalidator.ErrorAggregator;
+import org.openapitools.openapistylevalidator.custom.OpenAPIRule;
+import org.openapitools.openapistylevalidator.styleerror.StyleError;
+
+/**
+ * This rule will check whether tags are defined in the rule section
+ */
+public class CheckTagsAreDefinedRule extends OpenAPIRule {
```

Comment:
I love the idea of custom rules! Why did you put them in the CLI modules? I think they should go in the `lib` module so anyone could use them (cli, java, maven, gradle, etc...). But maybe there was a reason. Let me know!

Reply:
Yes.. you're it should not be there in the cli module. I just put it there to validate my design such that `loadAndExecuteRules()` method is able to find all the rules available in the class path.